docs/command_clu2adv.md

```diff
@@ -0,0 +1,196 @@
+# Convert mongodbatlas_cluster to mongodbatlas_advanced_cluster
+
+clusterToAdvancedCluster (clu2adv) command helps you migrate from `mongodbatlas_cluster` to `mongodbatlas_advanced_cluster` Provider 2.0.0 schema.
+
+This revised file migrates the Terraform configurations and state to the latest version and does not modify the resources deployed in MongoDB Atlas.
+
+You can find more information in the [Migration Guide: Cluster to Advanced Cluster](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/cluster-to-advanced-cluster-migration-guide).
+
+## Usage
+
+If you want to convert a Terraform configuration from `mongodbatlas_cluster` to `mongodbatlas_advanced_cluster`, use the following command:
+```bash
+atlas terraform clusterToAdvancedCluster --file in.tf --output out.tf
+```
+
+you can also use shorter aliases, e.g.: 
+```bash
+atlas tf clu2adv -f in.tf -o out.tf
+```
+
+### Command Options
+
+- `--file` or `-f`: Input file path containing the `mongodbatlas_cluster` configuration
+- `--output` or `-o`: Output file path for the converted `mongodbatlas_advanced_cluster` configuration
+- `--replaceOutput` or `-r`: Overwrite the output file if it exists, or even use the same output file as the input file
+- `--watch` or `-w`: Keep the plugin running and watching for changes in the input file
+- `--includeMoved` or `-m`: Include the `moved blocks` in the output file
+
+## Examples
+
+You can find [here](https://github.com/mongodb-labs/atlas-cli-plugin-terraform/tree/main/internal/convert/testdata/clu2adv) some examples of input files (suffix .in.tf) and the corresponding output files (suffix .out.tf).
+
+## Dynamic Blocks
+
+`dynamic` blocks are used to generate multiple nested blocks based on a set of values. 
+Given the different ways of using dynamic blocks, we recommend reviewing the output and making sure it fits your needs.
+
+### Dynamic blocks in tags and labels
+
+You can use `dynamic` blocks for `tags` and `labels`. The plugin assumes that `for_each` has an expression which is evaluated to a `map` of strings.
```

Comment:
```suggestion
You can use `dynamic` blocks for `tags` and `labels`. The plugin assumes that the value of `for_each` is an expression which evaluates to a `map` of strings.
```
Broadly speaking, an 'expression' is not something which one 'has'. Furthermore, a key like 'for_each' does not typically 'have' anything other than its value, which may be an expression, etc.